### PR TITLE
Automatic polling for exogenous signals

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
Thanks to this PR the `YarpRobotLoggerDevice` will automatically connect to the exogenous ports. So no need to run it before the walking-controller

Moreover: 
- The remote control board will use udp as a carrier
- the readouts from the genericSensorClient (wrenches) are retrieved using udp
- the multipleanalogsensorsclient data are collected with fast_tcp
- the logging of the robot cameras is disabled
- The files are formatted

### CODE REVIEW

**Review by isorrentino — APPROVED** [review]

File: devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp [hunk 1]
```diff
@@ -604,6 +652,80 @@ void YarpRobotLoggerDevice::unpackIMU(Eigen::Ref<const analog_sensor_t> signal,
...
+        }
+        for (const auto& [name, signal] : m_vectorSignals)
+        {
+            allPortsConnected = allPortsConnected && signal.connected;
+        }
+
+        if (!allPortsConnected)
+        {
+            // check for new messages
+            yarp::profiler::NetworkProfiler::getPortsList(yarpPorts);
```
Comment (S-Dafarra) [inline devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp:690]: Why do you need the full list of ports if you already know the name of the port to connect to?

Comment (GiulioRomualdi) [reply devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp:690]: to check if the port exists to establish the connection. maybe we can directly do the connection if not connected. I don't know if  we can improve the performances in this case

Comment (GiulioRomualdi) [reply devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp]: https://github.com/ami-iit/bipedal-locomotion-framework/pull/570/commits/05342243cffc707ee03c605ce8a7baa1b3328d69 should implement what you suggested

Comment (S-Dafarra) [reply devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp]: That's pretty neat!

**Review by S-Dafarra — APPROVED** [review]

**Pull request merged**